### DISCUSSION TITLE
[BUG] Scam!

### DISCUSSION THREAD

**Discussion by gromran** [Chia-Network/chia-blockchain#8813] [General]
since 7 month no one chia...
SCAM!

If you want to do something useful with your storage space where you get paid: use Storj

**Comment by maomingyang1314** [comment]
This was originally a capital stock operated by China's capital, pre-mined 32 million pieces, and it is estimated that the official has sold almost the same. Anyway, there is no third-party supervision.

**Comment by maomingyang1314** [comment]
Even the development of a mining pool agreement has to be delayed again and again, and the capabilities of this project team can be imagined. It's just a bunch of people who took the money to do things for others.

**Comment by maomingyang1314** [comment]
This is the consequence of capital speculation. Even pigs can fly to the sky. No need to care about this project anymore. There is no future. There is no technology but no team and no team.

**Comment by marssystems** [comment]
If you don't like this project, go and do something else.

**Comment by ByronAP** [comment]
Haahaaaaaa this is a funny thread

**Comment by rmica** [comment]
0 blocks in 7 months? Full-node alone doesn't win. Try adding some plots.

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - farming is based on luck and the ETW in the client is nothing more than an estimate, luck plays a factor and it is not uncommon to go 6x the ETW until winning. Chia has a strategic reserve of 21 million which is fully auditable https://docs.chia.net/custody/prefarm-audit-tutorial. Lastly, software development takes time to get correct, especially for an immutable ledger, however the pooling protocol is in full effect and can be reviewed in depth https://docs.chia.net/pool-protocol-specification .

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
